packages/utils/tokens/src/text-tokens.ts

```diff
@@ -14,14 +14,16 @@ export interface FontStyleTokens {
   fontWeight?: keyof Typography['weights'] | TextStyle['fontWeight'];
   fontLineHeight?: TextStyle['lineHeight'];
   fontLetterSpacing?: TextStyle['letterSpacing'];
-  fontDynamicTypeRamp?: string; // TODO(#2268): Import type from RN directly
+  // TODO(#2268): Import these from RN directly
+  fontDynamicTypeRamp?: string;
+  fontMaximumSize?: number;
```

Comment:
Can we add comments indicating these are Apple-specific component tokens?

Reply:
Can do.